libs/coin-near/src/api/archive-node-sdk.ts

```diff
@@ -215,7 +211,7 @@ export const getStakingPositions = async (
       const staked = new BigNumber(rawStaked);
       available = new BigNumber(available);
       pending = new BigNumber(pending);
-      rewards = new BigNumber(rawTotal).minus(deposit);
+      rewards = new BigNumber(rawTotal || "0").minus(staked);
```

Comment:
How are we calculating rewards now ? I don't understand since we removed rawTotal definition :thinking:

Reply:
L180 :smile: